test: Pass KERNEL env param in ginkgo-kernel job

brb opened the pull request:
Otherwise, the `RunsOnNetNextOr419Kernel()` returns `false` on the 4.19 build, which makes to skip some kube-proxy replacement related tests.

Also, disable the dynamic kernel vsn selection, as we currently run the job only on the 4.19 kernel.

Fixes: 72fe87afb80a ("test: Extend RunsOnNetNext helper family")

nebril commented on jenkinsfiles/ginkgo-kernel.Jenkinsfile:
```diff
@@ -175,3 +173,10 @@ pipeline {
 }
+
+def get_kernel(String default_kernel) {
+    dir("${TESTDIR}") {
```
Seems like this line causes the problem in CI.

nebril commented on jenkinsfiles/ginkgo-kernel.Jenkinsfile:
```diff
@@ -12,6 +12,9 @@ pipeline {
         TESTED_SUITE="k8s-${K8S_VERSION}"
         GINKGO_TIMEOUT="300m"
         DEFAULT_KERNEL="419"
+        KERNEL = get_kernel("${DEFAULT_KERNEL}")
```
If you set `KERNEL` here in a manner similar to https://github.com/cilium/cilium/blob/master/jenkinsfiles/ginkgo-gke.Jenkinsfile#L128 , it should be available as an env var in all children pipeline steps.

brb committed:
test: Pass KERNEL env param in ginkgo-kernel job

Otherwise, the RunsOnNetNextOr419Kernel() returns false on the 4.19
build, which makes to skip some kube-proxy replacement related tests.

Also, disable the dynamic kernel vsn selection, as we currently run the
job only on the 4.19 kernel.

Fixes: 72fe87afb80a ("test: Extend RunsOnNetNext helper family")
Signed-off-by: Martynas Pumputis <m@lambda.lt>

brb committed:
test: Disable debug mode when running on 4.19 kernel

On 4.19 kernel we need to disable debug mode, as otherwise the BPF
datapath programs are hitting the 4k verifier complexity limit when
the BPF NodePort is enabled.

Signed-off-by: Martynas Pumputis <m@lambda.lt>

brb committed:
WIP

Signed-off-by: Martynas Pumputis <m@lambda.lt>
